app/src/main/res/layout/download_dialog.xml

```diff
@@ -61,11 +61,24 @@
             android:text="@string/caption_setting_title"/>
     </RadioGroup>
 
+    <android.support.v7.widget.AppCompatCheckBox
+        android:id="@+id/dialog_download_checkbox_intelligent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/video_audio_group"
+        android:layout_marginBottom="6dp"
+        android:layout_marginLeft="20dp"
+        android:gravity="left"
```

Comment:
No need to set gravity here, it will mess up the alignment. It'll also cause issues with RTL languages.